src/test/ui/async-await/await-keyword/incorrect-syntax-suggestions.stderr

```diff
@@ -237,13 +237,14 @@ error[E0277]: the `?` operator can only be applied to values that implement `Try
   --> $DIR/incorrect-syntax-suggestions.rs:16:19
    |
 LL |     let _ = await bar()?;
```

Comment:
`await expr` seems not a valid syntax?

Reply:
It's indeed not. This is a pre-existing suggestion artifact that I would rather tackle in a separate PR.

Reply:
I just took a look: this is because `await bar()?` gets recovered as `bar()?.await` instead of `bar().await?`. Either one will be a pure guess. We could avoid the subsequent errors by _not_ recovering and just leaving an error node in the AST...

Reply:
I'll fix it in this PR after all.

Reply:
Done. We also silence the `await` in non-`async` `fn` error for this case in particular, but that should be fine.